doc/contribute/guidelines.rst

```diff
@@ -536,12 +536,21 @@ reference manuals, etc.
 Coding Style
 ============
 
+.. note::
+   Coding style is enforced on any new or modified code, but contributors are
+   not expected to correct the style on existing code that they are not
+   modifying.
+
 .. _Linux kernel coding style:
    https://kernel.org/doc/html/latest/process/coding-style.html
 
+.. _snake case:
+   https://en.wikipedia.org/wiki/Snake_case
+
 In general, follow the `Linux kernel coding style`_, with the following
-exceptions:
+exceptions and clarifications:
 
+* Use `snake case`_.
```

Comment:
I this valid ? What about macro's ?

Reply:
This is yet another reason why I avoided this statement in the first place. In the Linux Kernel coding style, this is not mentioned at all, and is instead implicit. I will add a clarification about macros.

Reply:
Can just change it to say snake case for variable and function names, therefore excluding defines and macros

Reply:
I clarified it a bit now.

Reply:
I think you can just remove it, what we want is covered by the Linux Kernel coding style.

Any extra rule will lead to more confusion:
* `txbuf` vs `tx_buf`, both should be ok,
* not sneak camel case macro's,